lib/models/StatsModel.js

```diff
@@ -0,0 +1,22 @@
+const { StatsClient } = require('arsenal').metrics;
+
+/**
+ * @class StatsModel
+ *
+ * @classdesc Extend and overwrite how timestamps are normalized by minutes
+ * rather than by seconds
+ */
+class StatsModel extends StatsClient {
+
+    /**
+    * normalize to the nearest interval
+    * @param {object} d - Date instance
+    * @return {number} timestamp - normalized to the nearest interval
+    */
+    _normalizeTimestamp(d) {
+        const m = d.getMinutes();
+        return d.setMinutes(m - m % (Math.floor(this._interval / 60)), 0, 0);
```

Comment:
Ideally we would make sure `this._interval` is greater than 60 to avoid doing a modulo by 0. No big deal though as it's just a validation issue.

Reply:
Ya I agree, I will need to think of a better way for validating the configuration options of interval and expiry to guarantee that the user will set these variables properly or handle the edge cases. For now, interval and expiry are always fixed.